### ISSUE TITLE
Get car image

### ISSUE THREAD

**Issue by kelyaenn** [Hyundai-Kia-Connect/kia_uvo#788]
Hello,

On the blue link app there's an image of the car showed on the Home screen
Is there an api endpoint that let us fetch that image ?

Thank you

**Comment by cdnninja** [comment]
Yes it exists.  Happy to receive pull requests to add this feature.

**Comment by kelyaenn** [comment]
I'm able to retrieve my car image url by listening to outgoing connection of my Bluelink app, but I can't find how to get it using the api.... 
I'll investigate further
